content/en/docs/reference/networking/virtual-ips.md

```diff
@@ -268,6 +271,24 @@ falls back to running in iptables proxy mode.
 
 {{< figure src="/images/docs/services-ipvs-overview.svg" title="Virtual IP address mechanism for Services, using IPVS mode" class="diagram-medium" >}}
 
+### `nftables` proxy mode {#proxy-mode-nftables}
+
+{{< feature-state for_k8s_version="v1.29" state="alpha" >}}
+
+_This proxy mode is only available on Linux nodes._
+
+The `nftables` proxy mode is similar in most ways to [the `iptables`
+mode](#proxy-mode-iptables), but uses the newer nftables kernel API
+rather than iptables. Currently the only major difference in
+functionality is that the `iptables` mode supports connecting to
+[NodePort Services](/docs/concepts/services-networking/service/#type-nodeport)
+on `127.0.0.1`, but the `nftables` mode does not.
```

Comment:
Try to describe this in a way that is valid even if we dropped the `iptables` proxy mode and it wasn't mentioned in documentation (or maybe just a stub to tell people it's no longer available).